### DISCUSSION TITLE
Access Filter Field in Visible Method

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#6133] [Help] [answered]
Is it possible to access the current value of a filter field from within a `visible()` method of a table column?

**Comment by intrepidws** [comment, marked as answer]
I figured it out:

```
Tables\Columns\TextColumn::make('recipient.full_name')
    ->visible(function (Livewire $livewire) {
        return $livewire->tableFilters['view']['is_inbox'];
    }),
```

**Reply by hainl2000** [reply]
what type of Livewire? I do same as you but have no function name 'tableFilters'.
<img width="747" alt="image" src="https://github.com/filamentphp/filament/assets/66208534/df13b3e1-ba15-4524-89d0-90109352f442">
